docs/tutorials/05-model-monitoring.ipynb

```diff
@@ -11,14 +11,16 @@
     "(realtime-monitor-drift-tutor)=\n",
```

Comment:
<p>this section can be remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/8410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>